pkg/time/mysql_time/mysql_time.go

```diff
@@ -17,6 +17,6 @@ package mysql_time
 import "time"
 
 func GetMysqlDefaultTime() *time.Time {
-	date := time.Date(1000, 01, 01, 0, 0, 0, 0, time.Local)
+	date := time.Date(1970, 01, 01, 0, 0, 0, 0, time.UTC)
```

Comment:
any other invoker will be influenced?

Reply:
Because the database specification requires a default time, the time zone of the previous default time is incorrect